### ISSUE TITLE
[Feature request] Jump to next Rest/Micro Break

### ISSUE THREAD

**Issue by Manni79** [rcaelers/workrave#374]
Hello,

I really miss a option to "jump to next break". Divided in jump to next Micro- or Rest-Break. For me this feature would be mandatory.
Hope to see it in the future.

Thanks a lot!

**Comment by kees-jan** [comment]
What would be the purpose of that option? I mean, if you want to take a break, you can simply stop working?

**Comment by Manni79** [comment]
E.g. when click on skip for a micro break and some minutes later want to start the micro break. 

Just "simply stop working" in my opinion is not the same. First this break would not count to statistics. But more important, the next micro countdown timer is running already if you just do a break without inform/start the break in Workrave. 
So, if you do not start the next micro break in Workrave, the next micro break will pop up some minutes later and not a full time range after your last micro break.

**Comment by kees-jan** [comment]
Hmm... You mean you have reading mode enabled?

**Comment by Manni79** [comment]
Yes, sorry, I am new to Workrave, so I have to learn what is important here.  
In my typical office day "read mode" is default. I very often read emails, have online meetings, read text and think about texts, look videos, read websites. 
So, "'natural breaks / auto-triggering-pauses" (smaller as a 30-90s) for me makes no sense. Because of this I use reading mode as default. 

But this is brining me to a new question. Is there a option to configure after what time a "'natural breaks / auto-triggering-pauses" is triggered? Default seems to be only a few seconds. So, the "'natural breaks / auto pause" without read mode triggers very, very often when I am working. But if I could set this time frame to 30-90s, then I would give it a try. I think in these longer time frame my mouse will be moved (or keyboard pressed) mostly.

**Comment by kees-jan** [comment]
Ah. Being in reading mode is an important clarification, thanks for that.

Both micro- and restbreaks have a duration. If Workrave considers you idle for that duration, then that'll count as a natural break. Personally, I have a micro-break duration of 30s, so if I'm idle for 30s, that counts as a natural micro-break (which will be in the statistics, by the way.

Workrave will indeed consider you idle after not doing anything for about 5 seconds. You'll see the green bar growing in the applet (or the workrave window), to see your break passing. However, the activity timer is not reset until you've taken the complete break. If you start working too early, the green bar disappears, and the activity timer resumes counting.

So to answer your question: If you set the duration of your breaks sufficiently high, you should be good to go.
